packages/compass-e2e-tests/helpers/selectors.ts

```diff
@@ -276,7 +276,7 @@ export const Single = {
 
 // Multiple Connections sidebar
 export const Multiple = {
-  ConnectionsTitle: '[data-testid="sidebar-header"]',
+  ConnectionsTitle: '[data-testid="connections-header"]',
```

Comment:
Took me a moment, but I realise now there's obviously no sidebar header in compass web, so te new selector makes more sense. 👍

Reply:
Yup! Similar to single connection, we render starting from the "connections" section of the sidebar. I took a look at the usage, and it seems like a safe switch (and the variable name still matches I think 😄)